jwst/jump/tests/test_jump_step.py

```diff
@@ -227,3 +227,22 @@ def test_two_CRs(generate_miri_reffiles, max_cores, setup_inputs):
         CR_group = next(CR_pool)
         assert (4 == np.max(out_model.groupdq[0, CR_group, CR_y_locs[i], CR_x_locs[i]]))
         assert (4 == np.max(out_model.groupdq[0, CR_group+8, CR_y_locs[i], CR_x_locs[i]]))
+
+
+@pytest.mark.parametrize("max_cores", MAXIMUM_CORES)
+def test_two_group_integration(generate_miri_reffiles, max_cores, setup_inputs):
+    override_gain, override_readnoise = generate_miri_reffiles
+    grouptime = 3.0
+    deltaDN = 5
+    ingain = 6
+    inreadnoise = np.float64(7)
+    ngroups = 2
+    CR_fraction = 5
+    xsize = 103
+    ysize = 102
+    model1, gdq, rnModel, pixdq, err, gain = setup_inputs(ngroups=ngroups,
+        nrows=ysize, ncols=xsize,
+        gain=ingain, readnoise=inreadnoise, deltatime=grouptime)
+    out_model = JumpStep.call(model1, override_gain=override_gain,
+                              override_readnoise=override_readnoise, maximum_cores=max_cores)
+    assert(out_model.meta.cal_step.jump == 'SKIPPED')
```

Comment:
It looks as if there is no newline at the end of this file.

Reply:
added the newline